docs/buildah-bud.md

```diff
@@ -0,0 +1,79 @@
+## buildah-bud "1" "April 2017" "buildah"
+
+## NAME
+buildah bud - Build an image using instructions from Dockerfiles.
+
+## SYNOPSIS
+**buildah** **bud | build-using-dockerfile** [*options* [...]] [**context**]
```

Comment:
If at least one of the Dockerfiles is a local file, we assume the directory that it's in is the context directory.